lib/api/controllers/collectionController.js

```diff
@@ -360,6 +361,23 @@ class CollectionController extends BaseController {
       .then(() => null);
   }
 
+  /**
+   * Deletes a collection
+   *
+   * @param {String} index
+   * @param {String} collection
+   *
+   * @returns {Promise}
+   */
+  delete (request) {
+    const { index, collection } = this.getIndexAndCollection(request);
+
+    return this.publicStorage.deleteCollection(index, collection)
+      .then(() => {
+        // return nothing
```

Comment:
Our current standard for acknowledged requests, but possibly not yet executed, is to return `{ "acknowledge": true }`

Here, if I understand correctly, we don't return until after the collection is deleted. So we could either return the acknowledgement message above, or simply `"OK"`, à la Redis.
Or, a bit more appropriate perhaps: `{ "deleted": true }`

The choice is yours.

But I think we shouldn't leave the "result" part null though: it can be puzzling for clients who are trying to decide if a response like that is an error or a success: `{error: null, result: null}`

Reply:
I prefer to return null in this case. If client want to know if the request was a success or not, they should check the `status` instead

Reply:
Fine by me.